### PR TITLE
check config in running kernel and return status[V2]

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>
v1 #1392

```
v2: modify method documentation
v2: added python enum

```

### CODE REVIEW

File: avocado/utils/linux_modules.py [hunk 1]
```diff
@@ -153,3 +165,31 @@ def module_is_loaded(module_name):
 def get_loaded_modules():
     lsmod_output = process.system_output('/sbin/lsmod').splitlines()[1:]
     return [line.split(None, 1)[0] for line in lsmod_output]
+
+
+def check_kernel_config(config_name):
+    """
+    Use a running kernel config file to  get config information
```
Comment (ldoktor) [inline avocado/utils/linux_modules.py]: nitpick, two spaces between `to` and `get` :wink: (no need to rebase, though)

File: avocado/utils/linux_modules.py [hunk 2]
```diff
@@ -153,3 +165,31 @@ def module_is_loaded(module_name):
...
+    :param  config_name:Name of kernel config to search
+    :type config_name:str
+    :return: Config status in running kernel
+    :rtype: enum
```
Comment (ldoktor) [inline avocado/utils/linux_modules.py]: Well the rtype is integer... But I don't think it's necessary to define it anyway...

File: avocado/utils/linux_modules.py [hunk 3]
```diff
@@ -153,3 +165,31 @@ def module_is_loaded(module_name):
 def get_loaded_modules():
     lsmod_output = process.system_output('/sbin/lsmod').splitlines()[1:]
     return [line.split(None, 1)[0] for line in lsmod_output]
+
+
+def check_kernel_config(config_name):
+    """
+    Use a running kernel config file to  get config information
+    :param  config_name:Name of kernel config to search
+    :type config_name:str
```
Comment (ldoktor) [inline avocado/utils/linux_modules.py]: I believe the space between `config_name:` and `str` is mandatory, but I'm not 100% sure.

**Comment by ldoktor** [comment]
Thanks for the fixes, it looks way better now. There are few little issues in the docstring so let me take care of them and merge it...

**Comment by ldoktor** [comment]
Updated, let me know whether it works for you and I'll merge it then (functionally it works well and correctly reports even values as builtin (which is IMO correct)

**Comment by PraveenPenguin** [comment]
@ldoktor Thanks !!

Regards
Praveen

**Pull request closed without merging**